Mathlib/RingTheory/Congruence.lean

```diff
@@ -223,10 +223,22 @@ end One
 
 section SMul
 
-variable [Add R] [MulOneClass R] [SMul α R] [IsScalarTower α R R] (c : RingCon R)
+variable [Add R] [Mul R]
+variable [SMul α R] [SMul β R] (c : RingCon R)
+variable [Con.CompatibleSMul α c.toCon] [Con.CompatibleSMul β c.toCon]
 
 instance : SMul α c.Quotient := inferInstanceAs (SMul α c.toCon.Quotient)
 
+instance [SMul α β] [IsScalarTower α β R] : IsScalarTower α β c.Quotient :=
+  inferInstanceAs (IsScalarTower α β c.toCon.Quotient)
+
+instance [SMulCommClass α β R] : SMulCommClass α β c.Quotient :=
+  inferInstanceAs (SMulCommClass α β c.toCon.Quotient)
+
+instance [SMul αᵐᵒᵖ R] [Con.CompatibleSMul αᵐᵒᵖ c.toCon] [IsCentralScalar α R] :
```

Comment:
```suggestion
instance [SMul αᵐᵒᵖ R] [c.toCon.CompatibleSMul αᵐᵒᵖ] [IsCentralScalar α R] :
```
or maybe even?
```suggestion
instance [SMul αᵐᵒᵖ R] [c.CompatibleSMul αᵐᵒᵖ] [IsCentralScalar α R] :
```

Reply:
`c.CompatibleSMul αᵐᵒᵖ` works, but I'm not a huge fan of how it swaps the argument order. Do you think it's worth doing anyway?

Reply:
I think it makes sense to the extent that `c` is meant to be fixed while `α` varies? I'll let you decide.